README.md

```diff
@@ -62,6 +62,18 @@ obs = env.reset()  # returns 'observation' concatenated to `desired_goal`
 action = policy_that_takes_in_vector(obs)
 ```
 
+## Registered environments
+This repository comes with a number of environments that are ready to be
+loaded via the `gym.make` interface. To do so, they must first be registered:
+
+```
+import multiworld
+import gym
+
+multiworld.register_all_envs()
```

Comment:
just wondering, would it be possible to put this in the `__.init__.py` of multiworld (so we don't have to explicitly register every time and can just do import multiworld, import gym. (Not sure if this goes totally against the spirit of the PR, but I think it wouldn't be too big of a deal to move the line there)

Reply:
specifically I meant the line `multiworld.register_all_envs()`

Reply:
Yes this goes against the spirit of the PR. Basically, the point of this PR is to make things explicit rather than implicit.

Reply:
What's the reason for preferring it to be explicit over implicit? I think Murtaza's suggestion is nice... Why go through the trouble of making the user call `register_all_envs` when `__init__` can do it for you?

Reply:
It's mainly a matter of readability. If someone sees this code:
```
import multiworld
import gym
env = gym.make('Sawyer-v0')
```
they might think, "Oh, that first import looks like an unnecessary import. Let me delete it." In fact, good IDEs will make this suggestion. However, this code:
```
import multiworld
import gym
multiworld.register_all_envs()
env = gym.make('Sawyer-v0')
```
makes it clear why you need to import multiworld and doesn't seem like that much of a hassle.

Reply:
Yeah actually i've seen that specific behavior many times with Pycharm. I think maybe explicit is for the best then.